pkg/k8s/client.go

```diff
@@ -45,6 +45,7 @@ import (
 var (
 	// ErrNilNode is returned when the Kubernetes API server has returned a nil node
 	ErrNilNode = goerrors.New("API server returned nil node")
+	client     kubernetes.Interface
```

Comment:
A comment explaining that one should use the Client() call to get this might be useful, along with some mention of when it might become non-nil. I realise you just moved it but it's now further away from the two functions that modify it so it isn't as apparent.

Reply:
Done.